### DISCUSSION TITLE
Working great on Pi 4

### DISCUSSION THREAD

**Discussion by krvw** [mikebrady/shairport-sync#1792] [General]
After trying a couple of cheap dongles, I found your software during a web search. Immediately bought a couple of Pi 4s (with 4 Gb RAM -- yes, overkill, but I'm an engineer damnit). Running shairport-sync and nqptp, I now have the whole-home audio I'd hoped for. (I use it for background music throughout my house, all day every day.)
I previously used a couple of Apple Airport Express routers, but they were discontinued, and AirPlay 2 kept going out of sync on them.
With the new setup, after a month of near-constant use, NOT ONCE has my music gone out of sync between rooms now.
One note: I found the AUX output on the Pi 4 wasn't really up to par, so I'm using a cheap USB audio dongle on each. $4.99 very well spent. The sound quality is better, and I have far better control over output volume.
So, that's a long winded way of saying **thank you for your work**. I'll be adding a few more of the Pi4s in other rooms shortly.
shairport-sync and nqptp are greatly appreciated.
Cheers,

Ken

**Comment by mikebrady** [comment]
Many thanks for your post. It's great to know other people are finding Shairport Sync useful!
